dev/js/wow_wizard.js

```diff
@@ -378,6 +378,30 @@
 			}
 		}
 
+		// If the step has steps depending on it, shows alert and clear the future steps.
+		function _clearFutureSteps($step) {
+			var depending_steps = [];
+			
+			for(var i = currentStep+1; i <= passedStepTracker; i++) {
+				var step_filled = wizard.settings.steps[i];
+				if(step_filled.isDependent) {
+					var found = false;
+					for(var j = 0; j < step_filled.steps.length && !found; j++) {
+						if(step_filled.steps[j].triggerStep == currentStep) {
+							depending_steps.push(i);
+							found = true;
+						}
+					}
+				}
+			}
+			if(depending_steps.length > 0) {
+				var message = (depending_steps.length == 1)?'Step ':'Steps ';
+				message += depending_steps + ' will change.'; 
+				alert(message);
+				passedStepTracker = depending_steps[0];
```

Comment:
This issue could be fixed adding a boolean parameter in settings like ```showAlerts``` and in the ``` _nextStep() ``` function check this parameter before call ``` _clearFutureSteps($step) ```